cmake/add_project_python_deps.cmake

```diff
@@ -118,9 +118,13 @@ if( VIAME_ENABLE_PYTORCH-NETHARN AND VIAME_ENABLE_GDAL )
 endif()
 
 if( VIAME_ENABLE_OPENCV OR VIAME_ENABLE_PYTORCH-NETHARN )
-  list( APPEND VIAME_PYTHON_BASIC_DEPS "ubelt<=1.2.3" "pygments" "bezier==2020.1.14" )
+  list( APPEND VIAME_PYTHON_BASIC_DEPS "ubelt==1.3.3" "pygments")
```

Comment:
Not sure if this will cause issues on Python 3.6, might need to select version based on python version but I can go back and do later if needed after attempted build with change

Reply:
I'm not quite sure how `VIAME_PYTHON_BASIC_DEPS` are being used, but if you are passing them to pip you can use environment markers so you don't have to think about version selection in CMake. E.g. In a Python 3.11 environment, If I run:

```bash
pip install "coverage>=6.1.1; python_version< '3.9' and python_version >= '3.8'"
```

It does nothing and prints the message:

```
Ignoring coverage: markers 'python_version < "3.9" and python_version >= "3.8"' don't match your environment
```


But it will install the package if I run:

```bash
pip install "coverage>=6.1.1; python_version < '3.12' and python_version >= '3.10'"    
```

However, ubelt up to version 1.3.6 still support Python 3.6, although I do plan to drop support for 3.6 and 3.7 in the next version.